### ISSUE TITLE
Update readers to accept .tif and .tiff files

### ISSUE THREAD

**Issue by dstansby** [brainglobe/brainglobe-napari-io#12]
@noisysky mentioned in https://github.com/brainglobe/brainglobe-napari-io/pull/9#discussion_r826242404 that the readers should probably be updated to accept both `.tif` and `.tiff` files (I presume these are the same, just a different extension 😄 )

**Comment by adamltyson** [comment]
Yup. Also see [comment](https://github.com/brainglobe/brainglobe-napari-io/pull/9/files/177bdaa7cb172b9c3b0399d2727bd5f1a242d8ed#r827818065) (although I'm probably not understanding the `npe2` API, having not used it yet).

**Comment by adamltyson** [comment]
I think this can be closed. This plugin doesn't load user-supplied data, but the outputs of other BrainGlobe tools.
